components/mpas-albany-landice/cime_config/buildlib

```diff
@@ -6,7 +6,7 @@ my $component = "mpas-albany-landice";
 my $component_dir = "glc";
 my $component_driver = "glc_cesm_driver";
 my $core = "landice";
-my $esm = "ACME";
+my $esm = "E3SM";
```

Comment:
@mark-petersen - these weren't in the previous PR?

Reply:
I have this pointing to a temporary branch e3sm/testing, so it's still showing previous diffs.  I'll rebase...

Reply:
Done.